boards/arm/lpcxpresso55s69/lpcxpresso55s69_cpu0.dts

```diff
@@ -86,31 +86,31 @@
 		#address-cells = <1>;
 		#size-cells = <1>;
 
-		boot_partition: partition@10000000 {
+		boot_partition: partition@0 {
```

Comment:
how is this related?

Reply:
It is not. My mistake.

Reply:
apparently `git rebase --randomly-combine-my-patches`